reinforcement_learning/examples/python/basic_usage.py

```diff
@@ -16,23 +16,31 @@ def main():
     model = rl_client.live_model(config, test_cb)
     model.init()
 
-    event_id = "event_id"
+    uuid = "event_id"
```

Comment:
why did you change the var name?

Reply:
Yes, let's just leave the original naming for clarity.